FirebaseMLModelDownloader/Sources/LocalModelInfo.swift

```diff
@@ -79,8 +75,9 @@ extension LocalModelInfo: DownloaderUserDefaultsWriteable {
 extension UserDefaults {
   static var firebaseMLDefaults: UserDefaults {
     let suiteName = "com.google.firebase.ml"
-    // TODO: reconsider force unwrapping
-    let defaults = UserDefaults(suiteName: suiteName)!
+    guard let defaults = UserDefaults(suiteName: suiteName) else {
+      return UserDefaults.standard
```

Comment:
@ryanwilson @maksymmalyhin How do you think I should handle this? I don't like falling back on the standard UserDefaults, but also feel that something like `fatalError` is too drastic.

Reply:
I think falling to standard user defaults is acceptable here.